iModelCore/ECDb/ECDb/SchemaManagerDispatcher.cpp

```diff
@@ -1641,6 +1646,60 @@ BentleyStatus MainSchemaManager::CanCreateOrUpdateRequiredTables() const
     return SUCCESS;
     }
 
+//---------------------------------------------------------------------------------------
+// @bsimethod
+//+---------------+---------------+---------------+---------------+---------------+------
+BentleyStatus MainSchemaManager::CheckForSelectWildCardLimit() const
+    {
+    ECDB_PERF_LOG_SCOPE("Schema import> Check for select wild card limit");
+    const int selectWildCardLimit = GetECDb().GetLimit(DbLimits::Column);
+    Utf8String sql = SqlPrintfString(R"x(
+        SELECT
+            [pm].[ClassId] [ClassId],
+            COUNT (*) + 2 [ColumnCount]
+        FROM
+            [ec_PropertyMap] [pm]
+            JOIN [ec_PropertyPath] [pp] ON [pp].[Id] = [pm].[PropertyPathId]
+        WHERE
+            [pp].[AccessString] NOT IN ('ECInstanceId', 'ECClassId')
+        GROUP BY [pm].[ClassId]
+        HAVING [ColumnCount] > %d)x", selectWildCardLimit).GetUtf8CP();

```

Comment:
use sql parameter and bind the limit using stmt.bindInt()

Reply:
Updated code to use SQL parameters and `stmt.BindInt()`